samples/openthread/ncp/README.rst

```diff
@@ -61,6 +85,21 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
+_ot_ncp_sample_features_enabling:
+
+Enabling additional features
+----------------------------
+
+When building the sample, you can enable the :ref:`additional sample features <ot_ncp_sample_features>`.
+To do this, run the build command with one additional parameter that defines the configuration files that you want to use.
+For example, to enable vendor hooks on the nRF52840 DK, run the following command:
+
+.. code-block:: console
+
+   west build -b nrf52840dk_nrf52840 -- -DCONF_FILE="prj.conf overlay-vendor_hook.conf"
```

Comment:
What about building with SES?
We're promoting SES as the first building method, but it cannot handle .cpp files.

FYI, @rlubos , @carlescufi .

Reply:
I think SES support is a part of specific task to provide it for all Thread samples and probably it will be developed later. Maybe such changes will be a good occasion to make also documentation update?

Reply:
Thanks for the update. Is there already a task for this SES support?
And yes, we can update the documentation then.

Reply:
Yes and I think it is planned to be done in this sprint.